Setup/UpgradeSchema.php

```diff
@@ -71,7 +70,66 @@ public function upgrade(SchemaSetupInterface $setup, ModuleContextInterface $con
                         null,
                         ['nullable' => false, 'default' => \Magento\Framework\DB\Ddl\Table::TIMESTAMP_INIT_UPDATE],
                         'Updated At'
-                    )->setComment('Post Table');
+                    )->setComment('Devstone Custom License Customer Table');
+                $installer->getConnection()->createTable($table);
+            }
+        }
+
+        //Add Max Usage
+        if (version_compare($context->getVersion(), '1.0.5', '<')) {
+            if (!$installer->tableExists(UsageSetup::ENTITY_TYPE_CODE . '_limit')) {
```

Comment:
Creating an extra table just to hold the `max_usage` field seems totally unnecessary and makes the code overly complex. The `max_usage` field could be added to the `devstone_usage_entity` table. Adding it as an EAV attribute would also be appropriate but creating another 1:1 table is just unnecessary and more work.